src/blog/how-anyone-can-start-testing-for-web-accessibility.md

```diff
@@ -4,7 +4,7 @@ author: Stephen Belyea
 title: How anyone can start testing for web accessibility
 featuredImage: ../images/woman-in-wheelchair.jpeg
 category: accessibility
-bio: Husband, father, habitual dork, front end developer + accessibility instigator, attempted writer, ex-pat Maritimer. 
+bio: Husband, father, habitual dork, front end developer + accessibility instigator, attempted writer, ex-pat Maritimer.
```

Comment:
Looks like some auto-formatting made it into the commit with this file. Let me know if you see any unwanted changes and I'll revert.